README.md

```diff
@@ -31,6 +31,9 @@ You'll need the following to run Reservations:
 * a [CAS](http://www.jasig.org/cas) authentication system (optional)
 
 ### Installation
+
+For Troubleshooting, see https://github.com/YaleSTC/wiki/wiki
```

Comment:
I don't think we need to capitalize "Troubleshooting". Also, let's use a Markdown link (see [here](https://guides.github.com/features/mastering-markdown/) for more info).